.circleci/config.yml

```diff
@@ -275,7 +274,7 @@ commands:
               echo "Deleting version $CURRENT_VERSION of service $SERVICE"
               gcloud app versions delete --quiet --service "$SERVICE" "$CURRENT_VERSION" --project "broad-ddp-${ENVIRONMENT}"
             done;
-            : # Need to make sure that exit code for step is a 0 even if versions delete failed
+            exit 0
 
```

Comment:
Not sure why this showing up here. This change already in 'parallel' branch.

Reply:
It might have been cleaner to split this off to a separate PR branched off of `develop` :)